.github/actions/check_artifact_size/action.yaml

```diff
@@ -0,0 +1,64 @@
+name: Check Artifact Size
+description: Check if the size of the newly created artifact specified by path is larger than the size of the previously uploaded artifact specified by name by the threshold.
+inputs:
+  workflow:
+    description: "Workflow to check artifact binary size for."
+    required: true
+  name:
+    description: "Name of the latest uploaded artifact."
+    required: true
+  path:
+    description: "Path to the artifact being checked."
+    required: true
+  threshold:
+    description: "Threshold for the increase in binary size of the artifact."
+    required: true
+runs:
+  using: "composite"
+  steps:
+      - name: Check Size
+        uses: actions/github-script@v6
+        with:
+          script: |
+            // Get the latest successful workflow run.
+            const workflowRuns = await github.rest.actions.listWorkflowRuns({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              workflow_id: '${{ inputs.workflow }}',
```

Comment:
Did you try to get the workflow name from `${{ github.workflow }}`? I think that would be better solution than to hardcode it in a param. If you can't access it in this context it definitely is accessible where this action is invoked (see comment below).

Reply:
Getting the workflow from ${{ github.workflow }} in evergreen.yaml instead.